{
  try {
    Thread.setDefaultUncaughtExceptionHandler(null);
    Assert.fail();
  }
 catch (  SecurityException se) {
  }
}
